README.md

```diff
@@ -249,6 +249,53 @@ the autocomplete/ctr file in your `.bashrc`, or manually like:
 $ source ./contrib/autocomplete/ctr
 ```
 
+### CRI
+
+`cri` is a [containerd](https://containerd.io/) plugin implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.proto).
+
+With it, you could run Kubernetes using containerd as the container runtime.
+![cri](./docs/cri.png)
+
+#### Current Status
+
+`cri` is a native plugin of containerd 1.1 and above. It is built into containerd and enabled by default.
```

Comment:
Suggested:

Since containerd 1.1, the `cri` plugin is built into the release binaries and enabled by default.